compiler/src/dmd/escape.d

```diff
@@ -1744,7 +1744,27 @@ void escapeByValue(Expression e, EscapeByResults* er, bool live = false, bool re
                     Parameter p = tf.parameterList[i - j];
                     const stc = tf.parameterStorageClass(null, p);
                     ScopeRef psr = buildScopeRef(stc);
-                    if (psr == ScopeRef.ReturnScope || psr == ScopeRef.Ref_ReturnScope)
```

Comment:
`Ref_ReturnScope` should not be treated different to `ReturnScope`

Reply:
they are different because a ReturnScope can be returned as a Ref